src/javascripts/helpers/data/visitorData/visitorsData.js

```diff
@@ -32,6 +32,20 @@ const deleteVisitor = (firebaseKey) => new Promise((resolve, reject) => {
   console.warn(firebaseKey);
 });
 
-// EDIT
+// SINGLE VISITOR
+const getSingleVisitor = (firebaseKey) => new Promise((resolve, reject) => {
+  axios.get(`${dbUrl}/Visitors/${firebaseKey}.json`)
+    .then((response) => resolve(response.data))
+    .catch((error) => reject(error));
+});
+
+// UPDATE
+const updateVisitor = (firebaseKey, visitorObject) => new Promise((resolve, reject) => {
+  axios.patch(`${dbUrl}/Visitors/${firebaseKey}.json`, visitorObject)
```

Comment:
I know your firebase is already set up so don't change anything, but it is good practice to have all your collections to be lowercase.

Reply:
Okay, thanks for the head's up - will remember that going forward!